PostHog/Autocapture/PostHogAutocaptureEventTracker.swift

```diff
@@ -205,6 +205,23 @@
             // first, call original method
             ph_swizzled_setContentOffset_Setter(contentOffset)
 
+            guard shouldTrack(self) else {
+                return
+            }
+
+            // ignore all keyboard events
+            if let window, window.isKeyboardWindow {
+                return
+            }
+
+            // nothing changed
+            guard
+                self.contentOffset.x.distance(to: contentOffset.x) > 0 ||
+                self.contentOffset.y.distance(to: contentOffset.y) > 0
```

Comment:
is this a workaround for detecting if the keyboard is open or closed?

Reply:
please describe with a comment in the code since it was not obvious to me only with the "nothing changed"